lib/preview/bucket.ex

```diff
@@ -69,4 +69,23 @@ defmodule Preview.Bucket do
     bucket = Application.get_env(:preview, :preview_bucket)
     Preview.Storage.delete_many(bucket, original_file_list -- new_file_list)
   end
+
+  def upload_index_sitemap(sitemap) do
+    upload_sitemap("sitemap", "sitemaps/sitemap.xml", sitemap)
+  end
+
+  def upload_package_sitemap(package, version, sitemap) do
+    upload_sitemap("sitemap/#{package}-#{version}", "sitemaps/#{package}-#{version}.xml", sitemap)
+  end
+
+  defp upload_sitemap(key, path, sitemap) do
+    opts = [
+      content_type: "text/xml",
+      cache_control: "public, max-age=3600",
+      meta: [{"surrogate-key", key}]
+    ]
```

Comment:
We don't use any of these options, right?

Reply:
good catch, we don't have a CDN in front. I'll remove all options besides content_type.

Reply:
`content_type` isn't needed either since it goes through `SitemapController` which will set it.